development/en-us/e2e-test.md

```diff
@@ -104,7 +104,7 @@ The goToTab method is provided in SecurityPage to test the corresponding sidebar
     }
 ```
 
-![SecurityPage](../../img/e2e-test/SecurityPage.png)
+![SecurityPage](../../img/e2e-test/SecurityPage_en.png)
```

Comment:
I think we should just add English version picture is better

Reply:
> I think we should just add English version picture is better

Do you mean that both Chinese and English choose to use the English version of the image?

Reply:
yeah， thatwhat i mean

Reply:
> yeah， thatwhat i mean

done : )